packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -35,7 +35,8 @@ dependencies:
 ''';
 
   @override
-  final String main = r'''
+  String get main => _main;
```

Comment:
Why bother with this?

Reply:
I was trying to figure out how to get around the @override lint for fields. I'll update this to not be convoluted like this.